src/route/dataset.route.js

```diff
@@ -0,0 +1,69 @@
+const {
+  koa,
+  errors: { NotFoundException }
+} = require('auth-middleware');
+const Router = require('koa-router');
+const datasets = require('../service/datasets');
+const configData = require('../data/config');
+const log = require('../log');
+
+class DatasetRouter {
```

Comment:
Just a style thing here. I don't like creating classes containing only static methods, for that I prefer inline objects. The reason behind that I that I expect classes to be instantiatable (you will use them via `new Blablabla()`), and when what you are doing is just grouping some functions inline objects make it clearer that you are now supposed to instantiate them.

I get that this still works though, so your call.

Reply:
@andres-arana about this, we just need to agree and do it the same everywhere :)